package.json

```diff
@@ -51,8 +57,8 @@
 		"postcss": "^8.3.4",
 		"proj4leaflet": "1.0.2",
 		"prop-types": "15.7.2",
-		"react": "^16.13.1",
-		"react-dom": "^16.13.1",
+		"react": "^17.0.2",
```

Comment:
It should be ^16.13.1, otherwise, I expect conflicts while install.

Not?

Reply:
It worked fine for me. Tested together with app-docs or app-emsn091Manila (both apps have v17 installed)

Reply:
Yes, I belive. But it needs React 0.17 in the App, where ptr-maps is used. What happens, if you use ptr-maps with "react": "^17.0.2" in the App with "react": "^16" ? Or we are starting with update process to react 0.17 in all packages? I have no problems with this, but I'm afraid that we will be forced to update all packages to the 0.17.

Reply:
As I said, I tested it in docs and manila app, where many ptr-packages with dependency to React 16 are used as well. And so far I haven't recognized any problem. Feel free to report any issue you will have with this combination.